Source/ACE/Entity/WorldObject.cs

```diff
@@ -606,21 +638,91 @@ protected WorldObject(ObjectGuid guid, AceObject aceObject)
             PaletteGuid = aceObject.PaletteId;
         }
 
-        public void SetCombatMode(CombatMode newCombatMode)
+        public void SetCombatMode(CombatMode newCombatMode, Player player)
```

Comment:
It will need to be moved to creature - they go into combat mode as well.   I just made a mistake and put this in the wrong object.